src/pages/LandingPage/LandingPage.jsx

```diff
@@ -1,17 +1,33 @@
 import React from "react";
-import { landing_image } from "assets/";
+import { landing_image, contact_image } from "assets/";
+import variables from "styles/colors.scss";
 import "./LandingPage.scss";
 
 const LandingPage = () => (
-  <div
-    className="w-full flex landing-page bg-cover bg-right"
-    style={{ backgroundImage: `url(${landing_image})` }}
-  >
-    <div className="p-6 self-end text-left">
-      <h1>Clarity Detailing Solutions</h1>
-      <h2>It's all in the details</h2>
+  <>
```

Comment:
Can't we use a div to wrap this? Not a big deal, just to avoid empty tags.

Reply:
Yep, resolved.